prov/sockets/src/sock_cntr.c

```diff
@@ -222,88 +222,107 @@ void sock_cntr_inc(struct sock_cntr *cntr)
+static int sock_cntr_set(struct fid_cntr *fid_cntr, uint64_t value)
 {
 	uint64_t new_val;
-	struct sock_cntr *_cntr;
+	struct sock_cntr *cntr;
 
-	_cntr = container_of(cntr, struct sock_cntr, cntr_fid);
-	pthread_mutex_lock(&_cntr->mut);
-	new_val = atomic_add(&_cntr->value, value);
-	atomic_set(&_cntr->last_read_val, new_val);
-	if(atomic_get(&_cntr->num_waiting))
-		pthread_cond_broadcast(&_cntr->cond);
-	pthread_mutex_unlock(&_cntr->mut);
-	sock_cntr_check_trigger_list(_cntr);
+	cntr = container_of(fid_cntr, struct sock_cntr, cntr_fid);
+	pthread_mutex_lock(&cntr->mut);
+	new_val = atomic_set(&cntr->value, value);
+	atomic_set(&cntr->last_read_val, new_val);
+	if(atomic_get(&cntr->num_waiting))
+		pthread_cond_broadcast(&cntr->cond);
+	pthread_mutex_unlock(&cntr->mut);
+	sock_cntr_check_trigger_list(cntr);
 	return 0;
 }
 
-static int sock_cntr_set(struct fid_cntr *cntr, uint64_t value)
+static int sock_cntr_adderr(struct fid_cntr *fid_cntr, uint64_t value)
 {
-	uint64_t new_val;
-	struct sock_cntr *_cntr;
+	struct sock_cntr *cntr;
 
-	_cntr = container_of(cntr, struct sock_cntr, cntr_fid);
-	pthread_mutex_lock(&_cntr->mut);
-	new_val = atomic_set(&_cntr->value, value);
-	atomic_set(&_cntr->last_read_val, new_val);
-	if(atomic_get(&_cntr->num_waiting))
-		pthread_cond_broadcast(&_cntr->cond);
-	pthread_mutex_unlock(&_cntr->mut);
-	sock_cntr_check_trigger_list(_cntr);
+	cntr = container_of(fid_cntr, struct sock_cntr, cntr_fid);
+	pthread_mutex_lock(&cntr->mut);
+	atomic_add(&cntr->err_cnt, value);
+	if (!cntr->err_flag)
+		cntr->err_flag = 1;
+	pthread_cond_signal(&cntr->cond);
+	pthread_mutex_unlock(&cntr->mut);
+	return 0;
+}
+
+static int sock_cntr_seterr(struct fid_cntr *fid_cntr, uint64_t value)
+{
+	struct sock_cntr *cntr;
+
+	cntr = container_of(fid_cntr, struct sock_cntr, cntr_fid);
+	pthread_mutex_lock(&cntr->mut);
+	atomic_set(&cntr->err_cnt, value);
+	if (!cntr->err_flag)
+		cntr->err_flag = 1;
+	pthread_cond_signal(&cntr->cond);
+	pthread_mutex_unlock(&cntr->mut);
 	return 0;
+
 }
 
-static int sock_cntr_wait(struct fid_cntr *cntr, uint64_t threshold,
+static int sock_cntr_wait(struct fid_cntr *fid_cntr, uint64_t threshold,
 			  int timeout)
 {
 	int last_read, ret = 0;
 	uint64_t start_ms = 0, end_ms = 0, remaining_ms = 0;
-	struct sock_cntr *_cntr;
+	struct sock_cntr *cntr;
 
-	_cntr = container_of(cntr, struct sock_cntr, cntr_fid);
-	pthread_mutex_lock(&_cntr->mut);
+	cntr = container_of(fid_cntr, struct sock_cntr, cntr_fid);
+	pthread_mutex_lock(&cntr->mut);
 
-	if (_cntr->err_flag) {
+	if (cntr->err_flag) {
 		ret = -FI_EAVAIL;
 		goto out;
 	}
 
-	if (atomic_get(&_cntr->value) >= threshold) {
+	if (atomic_get(&cntr->value) >= threshold) {
 		ret = 0;
 		goto out;
 	}
 
-	atomic_inc(&_cntr->num_waiting);
+	atomic_inc(&cntr->num_waiting);
 
 	if (timeout >= 0) {
 		start_ms = fi_gettime_ms();
 		end_ms = start_ms + timeout;
 	}
 
-        last_read = atomic_get(&_cntr->value);
+        last_read = atomic_get(&cntr->value);
```

Comment:
The original code uses spaces in place of tabs.  This just did a find/replace of the variable name to not use one that starts with an underscore.  I'll do a quick check of the indentation in the surrounding code.